2465.feature.rst

```diff
@@ -0,0 +1 @@
+ModelSim and Questa now support compilation into a named VHDL library ``lib`` using :make:var:`VHDL_SOURCES_<lib>`.
```

Comment:
This file needs to live in `documentation/source/newsfragments`.

Reply:
Ah, that's why it isn't showing up. I thought it was due to the previous PR's newsfragment getting screwed up. I'll still add that fix tonight.

Reply:
Oh huh, I swear it was in there before. Sorry about the troubles, I'll move it right now.

Reply:
Hmmyeah, earlier comments indicate it was in there. Must have botched something in the process. 

Anyway, suggestion applied, commits squashed. Should be good to go now!

Reply:
Ah, the linking to the make variable `VHDL_SOURCES_<lib>` doesn't work unfortunately, and the `<lib>` part isn't shown. Not sure how that would need to be quoted. :/
Maybe the linking isn't that important and we can just write ` ``VHDL_SOURCES_<lib>`` `.

Reply:
Yeah. I've changed it to ``VHDL_SOURCES_<lib>`` :)

Reply:
Well with the extra ` that is, but I can't get GitHub to cooperate.

Reply:
LGTM!